src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.round(p.lerp(a1, a2, amt) * colorModeA);
```

Comment:
Technical note: in JavaScript, rounding vs. bitcasting is still drastically slower, so the much faster way to round any `x` is to use `(x+0.5)|0`.